fnl/zest/macros.fnl

```diff
@@ -197,8 +202,9 @@
         va? (_dumb-varg? args)
         nargs (.. "-nargs=" (if va?  "*" (match len 0 "0" 1 "1" _ "*")))
         f-args (if va?  "<f-args>" (match len 0 "" 1 "<q-args>" _ "<f-args>"))
-        command (M.smart-concat ["command " nargs " " name " :call " `ZEST_VLUA# "(" f-args ")"])]
-    `(let [ZEST_VLUA# ,vlua]
+        vlua-sym (gensym :ZEST_VLUA)
+        command (M.smart-concat ["command " nargs " " name " :call " vlua-sym "(" f-args ")"])]
```

Comment:
I _think_ this'll work okay 🤔 I think it can just be turned into a string easily enough, but worth testing! I _did not check this_ since I don't really use zest myself. So I don't have any good config to test it against, I assume you will be able to do this easily 😄